src/main/java/com/luxoft/svinenkov/movieland/utils/JsonManualConverter.java

```diff
@@ -0,0 +1,35 @@
+package com.luxoft.svinenkov.movieland.utils;
+
+import com.luxoft.svinenkov.movieland.entity.Movie;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class JsonManualConverter {
```

Comment:
look at writeValueAsString in com.fasterxml.jackson.databind.ObjectMapper - it allows to do the same work in one row